codestyle/joda-time-forbidden-apis.txt

```diff
@@ -76,4 +76,7 @@ org.joda.time.chrono.ISOChronology#getInstance()
 
 @defaultMessage Doesn't handle edge cases where the start of day isn't midnight.
 org.joda.time.LocalDate#toDateTimeAtMidnight()
-org.joda.time.DateMidnight
\ No newline at end of file
+org.joda.time.DateMidnight
+
+com.google.common.io.BaseEncoding.base64
+javax.annotations.concurrent.GuardedBy @ Use com.google.errorprone.annotations.concurrent.GuardedBy
```

Comment:
This is a Joda-Time related list. Please add the exclusions to `codestyle/druid-forbidden-apis.txt` instead

Reply:
changed